dotnet/src/Microsoft.Agents.AI.Hosting.AzureFunctions/BuiltInFunctions.cs

```diff
@@ -83,12 +83,13 @@ public static async Task<HttpResponseData> RunAgentHttpAsync(
 
         string? threadIdValue = threadIdFromBody ?? threadIdFromQuery;
 
-        // If no session ID is provided, use a new one based on the function name and invocation ID.
-        // This may be better than a random one because it can be correlated with the function invocation.
-        // Specifying a session ID is how the caller correlates multiple calls to the same agent session.
+        // The thread_id is treated as a session key (not a full session ID).
```

Comment:
I reliazed the MCP tool reads thread id from an argument named `threadId`. Should we consider changing that to `thread_id`? 

https://github.com/microsoft/agent-framework/blob/132597957abab73e7fbe1393556457b64ae2f2b2/dotnet/src/Microsoft.Agents.AI.Hosting.AzureFunctions/BuiltInFunctions.cs#L164

Needs to be fixed on the metadata transformer as well.